core/api.go

```diff
@@ -304,10 +304,29 @@ type LoadBalancer interface {
 	Next() Member
 }
 
-// HazelcastJSON is a wrapper for json byte array.
+// HazelcastJSONValue is a wrapper for json byte array.
 // JsonString should be a valid json string.
-type HazelcastJSON struct {
-	JSONString []byte
+type HazelcastJSONValue struct {
+	jsonString []byte
+}
+
+func CreateHazelcastJSONValueFromString(jsonString string) *HazelcastJSONValue {
+	return &HazelcastJSONValue{[]byte(jsonString)}
+}
+func CreateHazelcastJSONValue(object interface{}) *HazelcastJSONValue {
+	if object == nil {
+		return nil
+	}
+	byteArray, _ := json.Marshal(object)
```

Comment:
what if this method returns an error? I think this function should return an error in that case.

Reply:
This slipped my attention. It seems only time marshal returns error is when an object is `function` `complex` or  `channel`